mobile/src/people/employees-list-item.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { TouchableHighlight, StyleSheet, Platform, Text, View, Image } from 'react-native';
+
+import { Avatar } from '../people/avatar';
+import { Employee } from '../reducers/organization/employee.model';
+import { employeesListItemStyles as styles } from './styles';
+
+interface EmployeesListItemProps {
+    employee: Employee;
+    id: string;
```

Comment:
where is it used?

Reply:
you mean id prop? It supposed to be used for further development as param for an employee selection -> navigation to its profile and picking up details by id.